analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -214,6 +170,59 @@ class Cargo(
             packageDir.startsWith(analysisRoot)
         } ?: false
 
+    private fun buildDependencyTree(
+        name: String,
+        version: String,
+        packages: Map<String, Package>,
+        metadata: JsonNode
+    ): PackageReference {
+        val node = metadata["packages"].single {
+            it["name"].textValue() == name && it["version"].textValue() == version
+        }
+        val dependencies = node["dependencies"]
+            .filter {
```

Comment:
Nit: This formatting style is new to our code base / maybe a but odd. So far we did put `.filter` etc. into the previous line. Similar below.

Reply:
Done.